### PR TITLE
NSS Void Raptor [MDB IGNORE] [SEMI-MODULAR]

### PR DESCRIPTION
**Pull request by HushTD**
## About The Pull Request

Adds a new map, the NSS Void Raptor. A single Z-level ship map inspired by Goonstation ships, Blueshift and Pubby. Its design is tighter and size is thinner, allowing for new experiences, more roleplay, and something different to the usual ship experience. I did hear that people liked Blueshift, but thought it is too large and laggy, as well as that we lacked more lower-pop maps, which is not uncommon, especially on my European hours, so I am hoping to kill two birds with one stone. Peep.

**Criticism, ideas, as well as all help, especially in atmospherics, maintenance and electricity departments are welcome,**

VoidRaptor-1-min

TODO:
- [x] Acquire more feedback and work on it
- [x] Move a few places around (ex. Psychiatrist and Morgue)
- [x] Make places less square
- [x] TEGU and Some Minor pets
- [x] Dragging Atmos and Cables everywhere
- [x] Dragging Disposals everywhere
- [x] Run the mapping checklist through every room
- [x] Get the large mining shuttle working
- [x] Furnishing and Decorating pass for rooms
- [x] Furnishing and Decorating pass for hallways
- [x] Fill in maintenance
- [x] External decoration pass
- [x] Playtests
- [x] Get the map test merged

POST LAUNCH:
-  Maintain the map and avoid complaints (impossible)
-  Derelict ships and ruins around the station and randomize them using the automapper
-  Remap borrowed areas like Atmospherics and the Engine
-  Look deeper into station security (which is deliberately weakened due to tightness for antagonists)

## How This Contributes To The Skyrat Roleplay Experience

As I said before, the station's tighter design should provide new roleplay experiences, as well as people constantly are requesting new maps, especially for lower population rounds, arguing about Kilo. While definitely it will not be a map for everyone, I believe from feedback it can be a satisfactory experience for roleplay and action.

## Changelog

:cl: Mahalia, Tegu Sprite by Aspev from Tegu Station.
add: A new ship, the NSS Void Raptor has just been launched from the dry dock and is looking for a crew to man it! Its tighter and thinner design is sure to provide new experiences and complaints.
add: The NSS Void Raptor comes with a small unique pet with its own terrarium in the Research Department.
/:cl:

### CODE REVIEW

**Review by Jolly-66 — COMMENTED** [review]
Please put [MDB IGNORE] in your PR title **BEFORE** you rectify the merge conflict, otherwise AA (the service provider of the bots) will be angry.

**Comment by QuacksQ** [comment]
Ngl the placement of T-comms and the way it looks makes it feel like an after thought. 
Atmos is also really small with very little project space to actually build in.
<details><summary>T-Comm Placement</summary>

image
</details>

**Comment by HushTD** [comment]
> Ngl the placement of T-comms and the way it looks makes it feel like an after thought. Atmos is also really small with very little project space to actually build in.

Atmos I am seeing if I can have someone a bit more competent to remap or some idea to do, there is space above for potential expansion, just unsure how to utilize it, am not an expert for it

The telecomms were a bit unplanned here, would the original idea of having them in the AI sat work? The grav gen could go here then.

**Review by Jolly-66 — COMMENTED** [review]
This is just a cursory review due to this being a draft (and as such, I treat drafts as a "work in progress"). However, you have asked me on the Discord for a review so this'll be as such. I did not open the map in the editor just as heads up.

<details>
<summary> Active Turfs </summary>

Active turfs are the bane of mapping, but relatively easy to rectify. They're simply amtos diffs at world initialization, which just means, you had an aired turf next to a space turf for example. These are the following locations that have issues:

Ordnance Lab.

image

R&D Server room.

image

Port EVA Shed.

image

SB EVA Shed.

image

</details>

<details>
<summary> Turf Stuff </summary>

Turfs near the station need to use `nearstation`, which give it full bright.

image

Make sure you're using the CORRECT external solars turf.

image

Funky turf stuff here.

image

image


</details>

<details>
<summary> Misc Stuff </summary>

Put the decals in the tanks. They're difficult to read under pipes.

image

Don't know if this is intentional but, space all this stuff out.

image

No donor/player references in the maps, please.

image

You don't need to do this with the wires. Just run them under the windows, and have them enter/exit as such.

image

</details>

As I said, this is just a very cursory review. I'll go more in depth once you get more done.

**Comment by jjpark-kb** [comment]
<details><summary>Hydro Trays</summary>

image
</details>

Aren't these the wrong hydroponic trays?

**Comment by jjpark-kb** [comment]
<details><summary>Awkward Cargo Hallway</summary>

image

</details>

This is a really awkward hallway in cargo.

**Comment by jjpark-kb** [comment]
<details><summary>Clown and Mime Connected</summary>

image
</details>

I feel the clown and mime room should be connected.

**Comment by Zonespace27** [comment]
Alright since mahalia asked me to review, going to go through this quickly by-dept
<details>
<summary>Things</summary>

General:

- No grav gen
- No lights
- Commissary shutters
image
- Hope this massive dead zone gets used
image


Service:

- I like the open kitchen/bar atrium, very fun to look at
- The glass on the table here seems a bit strange 
image
- This room seems a bit awkward, but maybe it won't be when it's decorated
image
- Chap's office is missing the soulshard
- Chap's place looks pretty nice

Supply:

- Two loader suits

Engi:

- what's this for?
image
- improper dir coolers
image

Sci:

- Robotics Couch
image
- Space tile ordinance
image

Med:

- Chem area looks a bit off, can't put my finger on it
image
- No viro monkey pen thindoor
image
- Chem storage is *really* far back
image
- Chem lab could use a table to the outside here
image


Command:
- AI upload should have thindoors protecting the boards
image
- NT decals clipping into APCs n stuff (not a fan of them in the first place tbh)
image

</details>

**Comment by HushTD** [comment]
> Ngl the placement of T-comms and the way it looks makes it feel like an after thought. Atmos is also really small with very little project space to actually build in.

Resolved, atmos hope to get someone to help me build a better one

> This is just a cursory review due to this being a draft (and as such, I treat drafts as a "work in progress"). However, you have asked me on the Discord for a review so this'll be as such. I did not open the map in the editor just as heads up.

All resolved or will be soon, I want to make a small maint dungeon gauntlet, will move surgeon elsewhere.

> Aren't these the wrong hydroponic trays?

Were correct ones with a changed icon, though I see the changed icon has some issues, returned to default

> This is a really awkward hallway in cargo.

Partially remapped, seeing about changing more soon

>I feel the clown and mime room should be connected.

Connected

>No grav gen

added preliminarily, need to re-area some places

>No Lights Shutters and such

Will come when I actually get decorating more

Rest is on toto and... can robotics not be a little cozy with a couch, was a small silly request I got, ha.

**Comment by jjpark-kb** [comment]
<details><summary>Cryo Location</summary>

image
</details>

I don't think Cryo should be located at arrivals, but rather closer to dorms.

**Comment by jjpark-kb** [comment]
<details><summary>Nuke Device</summary>

image
</details>

I know the nuke isn't used often, but there should be spaces all around it. I could only imagine someone blowing up the wall next to it and using the nuke machine and being unable to be targeted.

**Comment by jjpark-kb** [comment]
<details><summary>Armory and Labor Shuttle Space</summary>

image
</details>

So if the labor shuttle is down, this gives armory even more protection against space breaches. Should probably put at least a tile of space between the shuttle and armory.

**Comment by jjpark-kb** [comment]
<details><summary>Moving Cargo Guards</summary>

image
</details>

I think you could move the guard outpost for cargo to the bottom left of the picture, replacing a bit of the maint. This allows the guards access into maint, they are closer to the center, etc.. You could then wrap the QM's bed to there and put the mining vendors in front of it-- or something along those lines.

**Comment by HushTD** [comment]
More changes, most of comments should be resolved, rest will be soon... Updated screenshot in PR body, currently more remapping and piping. Also resolved conflict I guess for now.

**Comment by Jolly-66** [comment]
If it helps, you don't need to push every commit. You can simply COMMIT them, but you don't need to push them.
It'll "save" them until you eventually push them. This way, CI can run a tad more efficiently and won't get clogged. This is more of an issue when mass mirrors occur/mass merging, which can create issues if you're running to see if something is failing on CI.

**Comment by HushTD** [comment]
> If it helps, you don't need to push every commit. You can simply COMMIT them, but you don't need to push them. It'll "save" them until you eventually push them. This way, CI can run a tad more efficiently and won't get clogged. This is more of an issue when mass mirrors occur/mass merging, which can create issues if you're running to see if something is failing on CI.

I know how git works, I am mapping on two devices, and additionally well, if you do not push, there is no backup, ha. Sorry if I am making CI go brrr a bit, but I try only to make one commit a day unless I swap devices, that one bunch there was because I wanted linters to pass at least, maybe should make CI pass too.

**Comment by HushTD** [comment]
Updated checklists, got more feedback I will be working on, at least CI passes on the map, woo.

**Comment by HushTD** [comment]
Screenshot updated, thanks to all the tg folk, friends and more for helps, pointing out issues and minor testing.

**Comment by HushTD** [comment]
Okaaay, since all that is left is mostly polish or things I did not notice... as the saying goes, let us dance. Screen updated too.

**Review by Jolly-66 — CHANGES_REQUESTED** [review]
Here's somethings I've noticed.

<details>
<summary> Cameras </summary>

To preface, I know cameras can be disabled roundstart. So just double check these areas.

You're also missing critical cameras outside the AI sat. Please add them.

You also don't need to put cameras in all the heads bedrooms (barring the Captains). Its just a privacy thing. That and, I'm pretty sure some of our AI players probably don't wanna see the nasty.

You lack cameras in the solar control rooms. Please add them.

There's this blind spot in RnD.
image

Blind spot here in sec.
image

Here in the main hall.
image

This door here in med can be a problem if someone yells "AI door".
image

Streaks of blind spots in chemistry.
image

Blind spot here outside of chemistry.
image

Blind spot strip in departures.
image

Lack of camera here.
image

Tcomms blind spot.
image

Blind spot here in the HFR room.
image

For the atmos tanks, move the camera to the center back wall. This way the cameras can "see" forward.

Ditto for turbine.
image

Dead spot here.
image

Ditto.
image

Blind spot here in the prison.
image

Blind spot here in cargo.
image

</details>

<details>
<summary> Light/Light Switches </summary>

Please be mindful with how you placed light switches. They work based off of `area` defines, not that they're encased in a room. So, for example, a heads office may have a light switch in the office and their quarters, but due to the area define spanning both rooms, the light switch controls everything.

Please also double check your lights. Many of them are red, which, we don't need. It doesn't make sense in most rooms.

Also, the super matter room is incredibly red. Please fix that.

We don't need light switches in the halls.
image

Red light here.
image

Since this area is defined as `engine`, and lacks a light switch, its eternally dark. Either remove the light switches from the area or add one.
image

</details>

<details>
<summary>Other Nitpicks</summary>

Fix your area defines.
image

On every instance of the jets, you have them defined as an area. Use nearspace to give them `full bright`.
image

Two things here,
1. Please don't shift the machines onto the walls.
2. Change the brick walls to regular walls. It doesn't really look good meshing with regular walls.
image

This entire room is an absolute no.
1. Thats a real cult airlock. People who walk into it will be sent back and damaged. Use the friendly subtype.
2. Thats also a real rune. Yes, we don't run cultists, but I **do not** want that on any station in any caliber.
3. Runed metal. Get rid of it.
image

Pretty sure if a xeno spawns here, they'll get merked by the spiders. Not entirely sure since I don't know either of their defined factions off the top of my head.
image

Not too keen on letting people have access to guns. I'll leave that up to the others to decide.
Also, amount of space cash can be tweaked a bit (lowered).
image

Also, while testing, I wanted to obliterate the firebot. Please move him or remove him.
image

Improper uses of R-Walls:
image

image

Not too keen on the use of plasma glass/tables here. Please use regular glass windows, tables or reinforced tables.
image

Remove the rune trap, please. Only chaplains can remove them IIRC. Its relatively harmless, but people trying to enter the room will be stun locked for a few seconds. 
image

All of these event spawns will hog event spawns to favor this place, since it has 5 chances to spawn here.
image

Please don't var edit the SM.
image

Also not too keen on the floor here. Would much rather you just use regular reinforced flooring.
image

These walls need to be R-Walls
image

Unless things have changed upstream, I don't want a BZ canister to be obtainable. IIRC its required for RnD as part of an experiment. This instantly nullifies people having to create it (and is why XenoBio no longer starts with one).
image

The curtains here are mechanical, so players cannot open them by hand. Do you have a button set somewhere?
image

Swap this banner out for the mundane one. We can "inspire" people to heal them.
image

I would subtype out the floor due to its name and description.
You don't need to even replace this by hand, you could just find and replace, or write an update paths (just remember to ONLY PUSH your map and no other changes).
image

Change this to a maint door.
image

Not too much of a fan of the colored disposal chutes. Please revert them.

Also please be careful with your usage of double doors. Theres an issue with one where players can't bump into them properly.

</details>

With that all said and done, I would like to just state that, while we have the mapping verb to check for things like lights, air alarms, ect, you **do not** need to follow it to a T. Not every room will need a light switch, APC, ect. Its more so of just.. a subtle guide. It isn't perfect, but its a good idea to know what you may or may not need.

**Comment by HushTD** [comment]
Issues from Jolly and Kathrin so far should be resolved.

**Comment by tf-4** [comment]
<details>
<summary>Blueshield office</summary>

image
Remove the krav gloves
Not a huge fan of them getting a MOD tbh but not the end of the world.
</details>

 
<details>
<summary>Bridge</summary>

image
Remove the medkits, replace with one basic white one
Swap the advanced extinguisher for a regular one
Why the command headset?

image
Some misplaced areas here
</details>

<details>
<summary>Vault</summary>

The vault safe is a bit too stacked. Remove the briefcase.

image
The vault itself is also way too secure.
You have the HoP office one side, the HoS office the other, and two r-walls and electrified windows between it and space.
The only other map that has the vault not directly adjacent to space (icebox) only has one layer of windows and it's much farther from command / sec areas.
</details>

<details>
<summary>Warden office</summary>

image
Warden doesn't get a gun in their locker anymore
</details>

<details>
<summary>HoS office</summary>

image
Remove the gloves
</details>

<details>
<summary>Sec office</summary>

image
If they want tacklers they can buy them
</details>

<details>
<summary>Armoury</summary>

image
Remove the EMP kit
Needs a shotgun ammo locker with some lethal/rubbershot boxes
</details>

<details>
<summary>Perma space</summary>

image
These wirecutters have a toolspeed of 0.2 compared to the 1 of regular ones. Just use regular ones
</details>

<details>
<summary>Library</summary>

image
The tablet is a donator item
</details>

<details>
<summary>Dorms</summary>

image
Is this not just gonna break out every round?
</details>

<details>
<summary>Atmos</summary>

image
Atmos overall is uh. Pretty small. I get that it's basically just Kilo atmos, but Kilo atmos is one of the worse maps for atmos on account of it just being tiny.
This is just the atmos main in me wanting lots of space, so I'm not formally requesting this be made bigger. But it would be nice.

image
The turbine room feels really small considering a lot of space is being taken up by a roundstart-inaccessible flavour room.
Not a huge fan of the camera bug either.

image
I'd also open up this so that the turbine exterior is accessible from space. Makes it 100x easier to pipe space cooling.

image
Don't have distro/waste going out the top right door. With the air pipe there too it's taking up 3 out of 5 layers in the best place to pipe into the HFR. Ideally move air out of the way too.
</details>

<details>
<summary>Supermatter</summary>

Overall I think this is a neat and fairly unique design, but there are a few adjustments to be made.

image
Don't have the distro pipe going here, people like to put layer adaptors down and it will block them and be annoying.

image
You'll need a valve here before the passive vent if you want a temp gate into waste.

image
Not a fan of this injector, if people want to use an injector they should have to set it up themselves.

image
Atmos to SM line should pump in pre-filter, not post.
</details>

<details>
<summary>Maint robotics</summary>

image
Cute, but an intellicard isn't a great idea to have laying around.
</details>

**Comment by HushTD** [comment]
Anyhoww, to mention that feedback, resolved.

Things that were to be nerfed are nerfed, but for rest:

Blueshield - Gets even more gear on Blueshift, MOD included, it's a weak one.

Bridge - There are a few extra headsets in some places for roleplay and different cases, just like genetics has a free headset, moreso for RP and to incentivice some minor crime in some cases, it's an experiment.

Perma Space - Each Department has its own lizard with a piece of advanced tools, I feel a lil more space fun should be encouraged, wish I had something a little weaker, but experimental tools are not a full set and alien are even better.

Library - Wish that was marked better its a donor item, ree.

Dorms - Shork is tamed and a good boy, wish his name was not randomizing though.

Atmos - Issues fixed, potential better remap after playtests if I get some support.

SM - Fixed, Maint door added like on Delta

Maint Robotics - Tater placed in Tech Storage.

**Comment by Zonespace27** [comment]
<details>
<summary>Things</summary>
Armory lacks lighting

image

This robotics headset is unused- don't place it on maps
image

Having 2 pairs of sunglasses and a sec+ headset in the lawyer's office seems kinda sus to me ngl
image

Similar story with secmed
image

There are probably better places for the lustwish in the prison than this
image

Chapel transit tube can very easily get its wires cut by dust
image

Rotating these stasis beds in med would be nice
image

"Dull" shouldn't be capitalized
image

This lizarb seems a little too easy to get to
image


</details>

**Comment by jjpark-kb** [comment]
<details><summary>Missing Nearstation Areas</summary>

image

</details>

**Comment by jjpark-kb** [comment]
<details><summary>No Adv. Tools</summary>

image
image
image
image
image
image
image

</details>

**Comment by jjpark-kb** [comment]
<details><summary>Missed Lawyer/Court Opportunity</summary>

image
</details>
You could totally put a door between the courtroom and the lawyer.

**Comment by jjpark-kb** [comment]
<details><summary>Are you trying to kill new arrivals? hahaha</summary>

image
</details>

**Comment by jjpark-kb** [comment]
<details><summary>Oh no no no, not the cat surgeon!</summary>

image
</details>

**Comment by jjpark-kb** [comment]
<details><summary>Gas miners at their respective spots</summary>

image
</details>
I know we have the deployable ones, but thats really just to keep with modularity on the nonmodular maps... we can just have gas miners on this one.

**Comment by jjpark-kb** [comment]
<details><summary>intentional bridge sign?</summary>

image
</details>

**Comment by jjpark-kb** [comment]
<details><summary>Scary Situations</summary>

image
</details>
Imagine, bee simulator or control burn, right next to departure.

**Comment by HushTD** [comment]
For Zone, Solved

Armory lacks lighting - You missed the light switch

The too easy and unsupported headsets - Removed, some still remain for RP and minor crime, sunglasses reduced in count

Some Lizards made harder to access, requiring some creativity

Lustwish moved

Chapel has SMES now

Beds rotated, dagger normalized

**Comment by HushTD** [comment]
For JJPark

Nearstation fixed

Advanced tools are part of the larger overall easter lizard easter eggs as part of set, they are not really that strong, incentivize a little exploration and others seem fineish with it, I would use experimental tools but they are not a full set, I made some of them harder to get

Lawyer Court Door Added

Cat Surgeon is on Kilo and Blueshift, a lil maint mobs is good for guncargo and for medical to get a little more patching I feel, gotta earn the maints

Gas miners I was told not to place them, give atmos a little work with setting them

Bridge Sign gone, and the Shark is tamed and does not attack crew, I just thought the "Shark in an aquarium Aesthetic" is amazing, and got positive opinions on it.

For Holodeck we generally disallow these situations and programs as grief, but do not see an easyish solution to that, good to consider for future.

**Comment by GoldenAlpharex** [comment]
From a quick glance, after only really looking over the Command area.
<details><summary>HoP's Office</summary>
Why do you have the chair painted in green?
Also, you might want to have regular wood floors where I circled in red (and generally have a different floor color under doors, unless it's a floor sprite that doesn't update to take into account the other ones around it, so basically, no carpet under doors)

image
</details>
<details><summary>Captain's Office</summary>
I'll preface this one by saying that I think you should swap the Captain's office location with the Blueshield's and the Teleporter room that goes with it. It would have a lot less balance concerns, because then nobody would be able to sneakily break into the Captain's office with minimal chances of getting caught.
That being said, if you keep it here, I feel like double walls wouldn't be a bad idea, especially considering how close to the maintenance wall all the high-value items are on this map.
Furthermore... What's up with the Captain's locker? What did you add on top of it, and why is it necessary?
For the Hand teleporter, I'd suggest moving it behind the glass, as it would make more sense to do so, than making it look almost like a part of the scenery.

image

</details>
<details><summary>Blueshield's Office</summary>
Keep in mind this is without considering what I said in the previous comment about the Captain's office.
Why did you give him a combat Medkit? What's the gun case underneath it?
Why do they have another locker in there?
You should make the walls between the Teleporter room and the Blueshield's office reinforced.

image

</details>
<details><summary>AI Upload</summary>
I'd probably replace the middle window with a wall, and put a camera on it, unless Jolly or Kath say otherwise. It'd make sense to me, for a highly-secure location such as the Upload.
Also, what's up with this random Security holo-barrier?

image
image

</details>

**Comment by GoldenAlpharex** [comment]
> For Holodeck we generally disallow these situations and programs as grief, but do not see an easyish solution to that, good to consider for future.

You could simply have a "triple" wall, with a tile of space in the middle, kind of like how Delta does it for entering into the holodeck.

**Comment by HushTD** [comment]
@GoldenAlpharex 

HoP - I am keeping a theme of each Head getting a departamentally colored chair, and service kind of is green, but I can make it normal, just flavor, floors are an easy fix

Captain - I will see about mapping it in the past after playtests, it was mentioned in the past, I kind of am doing it a little like tram where it is disconnected, and it already is hard to break into places on such a tight map, so wanted to make it a little easier, Teleporter is an easy move, and the Armor is a reference to PubbyStation where the captain has the same armor.

The Blueshield - Gear room is based on Blueshift, this is the Blueshield's Medical Kit which is very weak and specifically defined in code, not a combat kit, I specifically colored it a little to differentiate it more, Rwall is easy. Gun kit is empty like on Blueshift too.

Upload - Feel the cam would see nothing and on the opposite wall would at least see something, but can add one. the sec Holobarrier is just a decoration for flavor, kind of "SECURE AREA, STAY AWAY" mood, can remove it.

Holodeck - Possibility, but it will require a severe cargo rethinking, which will take a lot of time, probably not before some tests.

**Comment by Vizoplez** [comment]
I saw this new map and simply thought.


Permabrig gonna be hella hard to suppress if traitors/arrested people decide to riot.

Due to how its been connected to the main station
image

This basically means that a traitor engineer, or really anyone trying to break out can sever these three connection lines (explosives, just spacing them, or by god-rodding it), forcing security to use EVA gear to get across, massively delaying response times and response /amounts/.

Just something to consider.

**Comment by Thrakerzod** [comment]
### I'm gonna leave the things I noticed that seem odd about Medical.

<details>
<summary>This is a bit long but primarily about Medical.</summary>

- Medical seems EXTREMELY long, the distance between front room and surgery/stasis rooms just seems like it's way too far when it comes to dealing with any patient's surgical needs.

- Stasis Room and Surgery Room being kitty corner from each other with a big room in-between with machinery centered is very annoying to get around when it comes to moving patients.

- On another note with that center room of Medical, Cryotubes almost _never_ get used, and really do not need a central position. I'd suggest maybe swapping the Cryotube room into the area with the Stasis beds, while making that central area kind of have four or so individual stasis bed rooms with tinted positional windows/windoors or something if you want to keep the central use of that center room. Atleast moving the Cryotubes to not that center room would allow that room to be used for something else.
- Surgery/Stasis rooms also being so far from Chem also is strange.
- Small thing but with surgeries usually taking place in Stasis beds, or even with the surgery beds, would there be any way to put plastic drapes around the individual stasis/surgery areas for more privacy, as they usually include the patient stripping for surgeries to be done.
- Quick note with water piping, if a Limb Grower is put on a tile with a water pipe under it will actually fill the Limb Grower with water. 
- Overall I'm not seeing any good ways of upgrading a medical setup with what is here, like a good place for SAD or the Limb Grower, along with any way to export things from the chemlab using chemical presses.
- In the front room of Medical, the bench up north could maybe be cut down to two bench tiles with a table next to it for the medkit to rest on instead of having it smack in the middle of the bench.
- Also in the front lobby area, there's a medical headset sitting on the tables behind the desk, with also a medical supplies vendor there too, while the front desk doesn't seem to be windoor'd or windowed, leaving anyone to just kinda hop the counter and do their own thing with those.
- The Crew Monitor sitting between the chairs also just seems out of place, with again anyone being able to just hop onto the table to look at anyone's position. Maybe moving it to the corner with the entertainment screen and fire alarm, but just having it there inbetween two chairs up against the desk seems off. 
- Maybe with the front desk, you can place the crew monitor inplace of a table there, and I dunno if you can actually do this, but put positional windows on the outside part of the desk to keep people from just hopping the counter.
- I assume those are tinted windows for the morgue but honestly I'm wondering why there's windows to the morgue at all. There's dead bodies there, no one wants to see those.
- I didn't get to look at the piping but from what it looks like, the the only output I see for the Mail Room leading to Medical just leads right into Morgue, kind of a strange place for mail to go.
- Actually I just noticed there's a chute behind the Cryotubes in the center area, honestly I just am really not vibing with the cryotubes in the center of the room.
- Another thing relating to cargo, I know it's connected to the Medical Storage Room, but the delivery location for Mules requiring the Mule to go across the station and _around_ the department entirely to deliver crates that way. Could maybe have it atleast deliver the crates to behind the front desk, removing that trash spawn in the maints to add the path.
- Psych office being way off in the back of Medical is a little funky, if you're taking patients you have to lead them through the entirety of Medical, past everything. Honestly if that shooting range below medical (which is a weird spot honestly as it's nowhere near dorms or the across the hallway activities room, usually "hangout spots") could be moved, I'd say put the psych office down there.
- Which leads me into a point that I'm kinda sick of being a trend and I see a comparison with this map, the Medical Breakroom. Almost always the butt end of Medical, usually never gets used due to the nature of medical being highly front-end, so leaving it always as a black box of not even RP use. Comparing this to the Science "Break Room" which is a central area of Science with nice wooden floors and a garden outside of it. Just seems like a bit of an insult. Bit of a rant but just a thing I'm sick of seeing.
</details>

<details>
<summary>Quick stuff about the rest of the map I noticed.</summary>

- HoP office is way off in the corner of the map, usually it's a central location since it's commonly used by the crew.
- Perma being a glass bridge being broken away from being spaced from the rest of Security asks for prison breaks.
- Kitchen seems extremely cramped with no room to move between two chefs without bumping into each other.
- Random question though but with the Chef getting CQC in areas labeled on the mapcode as "Kitchen" and "Bar," what sections of Service there would the Chef be able to kick a troublemaker's ass in? That entire center area? The Dining Area? The poker area?
- The relatively long walk between the ORM and the Mining Dock, involving likely many cargo techs bumping into miners trying to move their ore crates to the ORM.
- It seems like there's too many windows looking into places? I'll admit that I looked at other maps and noticed that there's windows leading into things like Secure Tech Storage, RD server room, command rooms, etc, but even after realizing that it still seemed like this map had seemingly windows opening paths specifically for Ninjas to see through and teleport into. Honestly I've revised this point a few times now and I'm realizing that that's prolly more intentional that I thought it was.
- Next antag thing, all of the maints seem crowded leading to specific areas for Security to corner people VERY easily, with not many ways to shake off people. All the maints seemingly go from hallways to departments, meaning either you need the access to the departments themselves to get through them, or you're trapped between a rock and a rock with a stunbaton.
- With another note being that it seems like almost every room has a maintenance access door. Every command personal room, the Science testing square thing all the way in the left of xenobio, Secure Tech Storage, the Psych office, department guards, etc.
</details>


I know this is a long post, but I had a conversation about it with some people and these were a lot of the things we brought up to each other, given that a majority of the medical stuff was from me. Reading it back, it's a lot of nitpicking I think, but just stuff we were throwing out to each other.

**Comment by HushTD** [comment]
Screenshot updated, Kind thanks to anyone who participated in the Test, especially these who reported issues.

File: modular_skyrat/modules/mapping/voidraptor/code/clothing.dm
Comment (GoldenAlpharex) [inline modular_skyrat/modules/mapping/voidraptor/code/clothing.dm]: You don't need Skyrat comments in modular files.
```suggestion
	new /obj/item/clothing/suit/toggle/labcoat/hospitalgown(src)
```

File: modular_skyrat/modules/mapping/voidraptor/code/mob.dm
Comment (GoldenAlpharex) [inline modular_skyrat/modules/mapping/voidraptor/code/mob.dm:95]: Amazing.

File: modular_skyrat/modules/mapping/voidraptor/code/turfs.dm
Comment (GoldenAlpharex) [inline modular_skyrat/modules/mapping/voidraptor/code/turfs.dm]: What's the point of this, exactly?

File: modular_skyrat/modules/mapping/voidraptor/code/mob.dm
Comment (GoldenAlpharex) [inline modular_skyrat/modules/mapping/voidraptor/code/mob.dm:1]: Don't we already have a file for modular pets like that one Botany bee?

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/mapping/voidraptor/code/mob.dm
Comment (HushTD) [reply modular_skyrat/modules/mapping/voidraptor/code/mob.dm:1]: Was told to put the code out of master files, so here it goes, unless I should return it.

File: modular_skyrat/modules/mapping/voidraptor/code/turfs.dm
Comment (HushTD) [reply modular_skyrat/modules/mapping/voidraptor/code/turfs.dm]: So it has a normal name instead of weird one, no description and drops a floor tile, I can just make it a subtype of iron floor if you prefer, does not make a difference.

File: modular_skyrat/modules/mapping/voidraptor/code/mob.dm
Comment (HushTD) [reply modular_skyrat/modules/mapping/voidraptor/code/mob.dm:95]: Kentucky from kilo actually subtyped and renamed

File: modular_skyrat/modules/mapping/voidraptor/code/clothing.dm
Comment (HushTD) [reply modular_skyrat/modules/mapping/voidraptor/code/clothing.dm]: Lil too much copypasta, gonna fix

File: modular_skyrat/modules/mapping/voidraptor/code/turfs.dm
Comment (GoldenAlpharex) [reply modular_skyrat/modules/mapping/voidraptor/code/turfs.dm]: Yeah but like, why does it need to not have a name, if it's the standard for vaults to be like that?

Comment (HushTD) [reply modular_skyrat/modules/mapping/voidraptor/code/turfs.dm]: The normal one is used sparsely on maps and called "strange floor" with a desc "You feel a strange nostalgia from looking at this..."

I have the whole command area plated with it for the unique sprite and look. Its just a normal floor otherwise and this returns the name and desc to such, also makes the tile not poof.

Comment (HushTD) [reply modular_skyrat/modules/mapping/voidraptor/code/turfs.dm]: Floor removed and some places lil remapped

**Review by Jolly-66 — CHANGES_REQUESTED** [review]
I lost my review because I accidentally closed my tab.

These were my nitpicks:

- Carp spawns outside need to be placed better. During testing carps broke into the station.
- Area defines north east near HoP line need to be fixed, specifically the atmos maint and power station room. Also, the power monitor in that room is useless. Only that room APC is on the circuit.
- Cold room in medical is too cold. Use the freezer from the kitchen, or have it start off.
- Move the cameras in the court room a bit off the lights. We want to avoid wall clutter as much as possible.
- The bottom button in the bar lacks req_access.
- The inner windows bordering the HoP office from the line need to be electrified.
- The burn tank in ordnance is a blind spot.
- This is just my stylistic nitpick, but grass under **full tile** windows looks off. Panes look better, but thats just a me thing.
- Remove the duplicate Donk Soft vendors from anywhere except dorms.
- External cameras don't need area defines they'll work just fine.
- The Holy signs are not directional, fix that (and double check any of those signs as well).
- Room security for certain rooms needs touching up, specifically in science. If a room borders a hall or maint that anyone can access, the bordering walls need to be r-walls.
- I would probably also make a custom area define for the pizza place.
- Also, for every little airlock you have that has tiny fans, remove the tiny fans and put cycling airlocks there.
- All the solar controller rooms lack cameras, please fix that.
- Also, external airlocks **do not** need external access helpers. They have a unique var known as `space_dir`. This'll follow thw `1, 2, 4, 8` dir method we use. The dir MUST be facing space.

For more information on `space_dir` you can look at `airlock_types.dm` ln 336.
Also, heres the comment:
```
/// Whether or not the airlock can be opened without access from a certain direction while powered, or with bare hands from any direction while unpowered OR pressurized.
	var/space_dir = null
```

**Comment by HushTD** [comment]
Most if not all should be resolved, will look about checking for dirs, walls/rwalls and a lil more polish.

**Comment by Tastyfish** [comment]
All things that use fluid pipes connect directionally (like atmos machines).

image

Thus, this sink and shower just connect to each other and not the duct next to them. The airlock room either needs to be 3x3 with a fluid duct in the middle, or they need to be otherwise rearranged the other direction. This applies to both science entrances, and possibly other places I haven't noticed.

**Comment by gamermomentum** [comment]
Put some doors where these firelocks are. If not all of them, at least for the locker room. 
image

Also I'm not gonna chase them around the whole map for screenshots but mapped in advanced tools are really quirky and probably shouldn't be scattered around as static loot. 
Also the botany chem dispenser roundstart might be a little much. 

I haven't actually played the map yet as a station character but I saw while observing that these maint doors just don't have an access requirement. Is that deliberate?
image

**Comment by HushTD** [comment]
@gamermomentum

Updated the screenshot, the doors were already added in a PR, just screenshot was not updated
The doors used to be maint access, but now require sec in recent PRs too. I am looking for a way to give more around-station maint flow, but see that might be impossible there, I generally need to add more maintenance

**Comment by HushTD** [comment]
@Tastyfish resolved this one too and will be on lookout for future instances.

**Review by GoldenAlpharex — APPROVED** [review]
We've been testing it for long enough, I think it's due for a merge now.

**Pull request merged**
